### ISSUE TITLE
Hide wrongly detected DATABASE on Heroku

### ISSUE THREAD

**Issue by tbouge-steeple** [pganalyze/collector#344]
Hello,

Thank you for your work.

I have one of our add-on that add a ENV on our running collector; this env is named `HEROKU_METRICS_URL`. The problem is that the collector handle it as a PG URL and try to connect to it. There is no immediate problem with this kind of behavior, but there are two glitches. 
- The collector adds a DB named `HEROKU_METRICS` on PGAnalyze website 
- when I run `collector --test` there is some log saying that he can't connect to this DB, as it's an HTTPS URI: (`https://app.metrics.heroku.com/[...REDACTED...]`)

There are 2 solutions. Add a denylist (or allowlist) OR detect that the URL isn't a pg schema. 

```go
`postgres(?:ql)?.*\`gm
```

**Comment by lfittl** [comment]
@tbouge-steeple Thanks for bringing this up!

I think if we can avoid it, it would be great not to add more configuration settings.

For this specific issue, maybe detecting that a URL isn't a Postgres schema would be good enough? It should be fairly straightforward to add to the logic here: https://github.com/pganalyze/collector/blob/main/config/read.go#L538

Would you be interested in making a PR that adds this? (otherwise I'm also happy to tackle it, but happy about contributions of course)

**Comment by msakrejda** [comment]
Fixed by #352!
